### PR TITLE
Feat : #5057 UI : Add support for adding a custom field to an entity

### LINKED ISSUE #5057: UI : Add support for adding a field to an entity type.
Refer #4920

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
Fixes #5057 
### Describe your changes :

I worked on adding support for a custom entity page from where users can add custom fields to an entity.

#
### Type of change :

- [x] New feature

#
### Frontend Preview (Screenshots) :
image

image

image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

**Comment by Sachin-chaurasiya** [comment]
Blocked by #5042

File: openmetadata-ui/src/main/resources/ui/src/components/CustomEntityDetail/AddCustomField/RightPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      <h6 className="tw-heading tw-text-base" data-testid="header">
+        Lorem, ipsum.
+      </h6>
+      <div className="tw-mb-5" data-testid="body">
+        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Adipisci ab,
+        rem molestiae culpa iure fugit cumque vel, nisi quam sed reprehenderit
+        exercitationem assumenda odio doloribus corrupti magni perferendis
+        veniam minus porro. Itaque accusamus veritatis esse laborum odit, nobis
+        sit quibusdam modi dolores iusto voluptas labore excepturi, tempora
+        praesentium nemo voluptatem?
```
Comment (shahsank3t) [inline openmetadata-ui/src/main/resources/ui/src/components/CustomEntityDetail/AddCustomField/RightPanel.tsx:28]: Can we add some meaningful information here?

Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/components/CustomEntityDetail/AddCustomField/RightPanel.tsx:28]: @shahsank3t, Sure. I will sync up with @ShilpaVernekar and replace this lorem lipsum with actual information
.

**Comment by ShilpaVernekar** [comment]
@Sachin-chaurasiya - Here's the content:
**Add a Custom Field**
OpenMetadata supports custom fields in the Table entity. Create a custom field by adding a unique field name. The name must start with a lowercase letter, as preferred in the camelCase format. Uppercase letters and numbers can be included in the field name; but spaces, underscores, and dots are not supported. Select the preferred field Type from among the options provided. Describe your custom field to provide more information to your team.

**Comment by harshach** [comment]
@Sachin-chaurasiya are you doing the validation on the space , dots and _ etc.. when the user is trying to add a field

**Comment by Sachin-chaurasiya** [comment]
> @Sachin-chaurasiya are you doing the validation on the space , dots and _ etc.. when the user is trying to add a field

@harshach , Yes.

**Review by vivekratnavel — APPROVED** [review]

**Review by shahsank3t — APPROVED** [review]

**Pull request merged**